docs/build-the-docs.md

```diff
@@ -19,21 +19,53 @@ docker run -ti --rm \
     -v $(pwd):$(pwd) \
     -w $(pwd) \
     antora/antora:1.0.1 \
-    generate site.yml
+    generate \
+        --clean \
+        --pull \
+        --quiet \
+        --silent \
+        --ui-bundle-url <url-or-path-to-ui-bundle>i \
+        --url <site.url> \
+        site.yml
 ```
+This build command configuration:
 
-This command starts up the Antora Docker container, removes the existing documentation, and then runs Antora's generate command, an regenerates the documentation.
-If all goes well, you will _not_ see any console output.
+- Suppresses all output (`quiet` and `silent`)
+- Remove the output directory before publishing the site (`clean`)
+- Download updates to remote resources (`pull`)
```

Comment:
Wouldn´t it be correct to say 
``Download updates to remote resources`` --> ``Download updates from remote resources``

Reply:
Yes. Well spotted.